--- LCEVCdec-4.0.5/CMakeLists.txt.orig	2026-02-04 13:09:57.000000000 +0100
+++ LCEVCdec-4.0.5/CMakeLists.txt	2026-02-26 20:47:03.863226363 +0100
@@ -170,11 +170,9 @@ if (EMSCRIPTEN)
 endif ()
 
 # Utility
-if (VN_SDK_EXECUTABLES OR VN_SDK_UNIT_TESTS)
     lcevc_add_subdirectory(src/utility)
     lcevc_add_subdirectory_if(src/utility/test/unit VN_SDK_UNIT_TESTS)
     lcevc_add_subdirectory_if(src/utility/test/utilities VN_SDK_UNIT_TESTS)
-endif ()
 
 lcevc_add_subdirectory_if(docs/sphinx VN_SDK_DOCS)
 
