--- LCEVCdec-4.0.5/src/legacy/decoder/CMakeLists.txt.orig	2026-02-04 13:09:57.000000000 +0100
+++ LCEVCdec-4.0.5/src/legacy/decoder/CMakeLists.txt	2026-02-27 17:46:18.440578730 +0100
@@ -26,7 +26,7 @@ include("Sources.cmake")
 # -------------------------------------------------------------------------------------------------
 
 set(PUBLIC_LIBRARIES lcevc_dec::platform lcevc_dec::compiler)
-set(PRIVATE_LIBRARIES lcevc_dec::core_version)
+set(PRIVATE_LIBRARIES lcevc_dec::core_version m)
 
 # Explicit static library for legacy unit tests
 # -------------------------------------------------------------------------------------------------
